### ISSUE TITLE
Downloading parallelization

### ISSUE THREAD

**Issue by emthompson-usgs** [usgs/groundmotion-processing#306]
Look into whether or not obspy's mass downloader already doing some form of parallelization? If so, are there any ways to configure this? If not, should we fork the download requests ourselves? It seems like they are threading up to 3 processes per data center, but we could potentially parallelize across data center. Of course, this might be of little value if it gets throttled by connection bandwidth.

**Comment by emthompson-usgs** [comment]
@jas82 Can you look into this?

**Comment by jas82** [comment]
These two parameters are used by obspy's mass downloader and could potentially speed up that process:

- chunk_size_in_mb (default is 25 MB which is pretty small)

- threads_per_client (default is 3, however I noted that this package uses 5 threads per client for an fdsn fetcher): https://www.seiscomp.de/seiscomp3/doc/applications/fdsnws_fetch.html

I downloaded data for South Napa earthquake with 3 and 5 threads and I got slightly less records when I used 5 threads per client. I can try and run a profiler. However this will probably just showcase my slowish internet speed and doesn't seem like a good way to determine optimum parameters. It sounds like these parameters require communicating with the appropriate data provider to make sure no abuse is taking place.
